doc/sphinx/practical-tools/utilities.rst

```diff
@@ -145,7 +145,9 @@ An optional file ``CoqMakefile.local`` can be provided by the user in order to
 extend ``CoqMakefile``. In particular one can declare custom actions to be
 performed before or after the build process. Similarly one can customize the
 install target or even provide new targets. Extension points are documented in
-paragraph :ref:`coqmakefilelocal`.
+paragraph :ref:`coqmakefilelocal`. If the user needs to access variables
+defined in the ``Makefile``, the optional file ``CoqMakefile.local-late`` can
```

Comment:
I'd like to a have a *non exhaustive* list of variables which are accessible, a bit like the ones below. Ideally one should not read the generated file in order to write the .local one.

Reply:
I've added some more documentation, please take a look and let me know what you think.